PhuongProject/ArrayCalculation.cs

```diff
@@ -20,10 +20,109 @@ public static int[] ArrayCalculate(int[] array1, int[] array2)
             int l2 = array2.Length;
             for (int i = 0; i < l2; i++)
             {
-               array1[array2[i]] += i;
+                array1[array2[i]] += i;
 
             }
             return array1;
         }
+
+        /// <summary>
+        /// Return the first 10 numbers of sum of all elements in an array
+        /// </summary>
+        /// <param name="array">The array</param>
+        /// <returns>The first 10 numbers of sum of all elements in an array</returns>
+        public static string FirstTenArraySum(string[] array)
+        {
+            int l = array.Length;
+
+            string[] cutString = new string[l];
+            for (int i = 0; i < l; i++)
+            {
+                cutString[i] = array[i].Substring(0, 15);
```

Comment:
why 15?

Reply:
Because ulong can hold up to 15 digit. The more number of digit in this case, the more accurate result returned. If I sum 12 digit numbers and get the first 10 digit, the probability of the incorrect result is 1/100. If I sum 15 digit numbers and get the  first 10 digit, the probability of the incorrect result is 1/100000